test/run-drun/ok/pass-class-self.diff-low.ok

```diff
@@ -1,4 +1,5 @@
 --- pass-class-self.run
 +++ pass-class-self.run-low
-@@ -0,0 +1 @@
+@@ -0,0 +1,2 @@
 +Before!
++ys6dh-5cjiq-5dc
```

Comment:
Why this diff? Didn't `run.ok` also change, did it?

Reply:
Good question...

Reply:
Hmm, why is the having any side-effect at all? The test is an actor class declaration and should do nothing until instantiated.

Reply:
Yeah, it is probably doing what `drun` does, auto-deploying.

Reply:
Run doesn't instantiate the class, but run-ir etc do.

I tried this on master (removing the self references) and it seems to be existing behaviour. Somewhat odd though, but perhaps just an artifact of "compiled" code.

Reply:
Ah, the ir_interpreter decides to execute the body of an actor class with zero arguments (to match semantics of installation on ic).

Nvm.

Reply:
It helps to see the side effects!